commit 7bbed91df2ef066e7897e589b62867ac27f3ff2b (refs/changes/46/4546/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-07-19T17:14:31-07:00 (1 year, 3 months ago)
    
    joyent/node-triton#249 Error when creating or deleting profiles when using node v10
    Reviewed by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 1b3d4c3..501276a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -6,6 +6,11 @@ Known issues:
 
 ## not yet released
 
+## 6.1.2
+
+- [joyent/node-triton#249] Error when creating or deleting profiles when
+  using node v10.
+
 ## 6.1.1
 
 - [TRITON-598] Fix error handling for `triton network get-default` when
diff --git a/lib/common.js b/lib/common.js
index 73051e3..609d0a2 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -618,9 +618,9 @@ function promptYesNo(opts_, cb) {
     stdin.on('data', onData);
 
     function postInput() {
+        stdout.write('\n');
         stdin.setRawMode(false);
         stdin.pause();
-        stdin.write('\n');
         stdin.removeListener('data', onData);
     }
 
diff --git a/package.json b/package.json
index 337c993..909b0af 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "6.1.1",
+  "version": "6.1.2",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
